src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/miner/MiningTask.java

```diff
@@ -268,8 +267,8 @@ private boolean push(@Nonnull ItemStack item) {
             if (chest.getType() == Material.CHEST) {
                 BlockState state = PaperLib.getBlockState(chest, false).getState();
 
-                if (state instanceof Chest) {
-                    Inventory inv = ((Chest) state).getBlockInventory();
+                if (state instanceof Chest chestState) {
```

Comment:
I think this may be a conflict that was resolved as my PR was merged in but I had called all of these patterns `instanceof Chest chest` so perhaps either change this to chest or change the same patterns in `VanillaInventoryDropHandler` and `OutputChest` ?

Reply:
I renamed this because this class already has a field named ´chest´

Reply:
Quite right, didn't pay enough attention ha